webapp/src/components/Modals/BuyWithCryptoModal/BuyWithCryptoModal.tsx

```diff
@@ -340,10 +340,16 @@ export const BuyWithCryptoModal = (props: Props) => {
     )
   }, [isFetchingBalance, isBuyingAsset, asset.chainId, isLoadingAuthorization, onBuyWithCard, handleBuyWithCard, onGetMana, onClose])
 
+  const onPayWithCredits = useCallback(() => {
+    if (onBuyWithCredits) {
+      onBuyWithCredits(BigInt(finalPrice))
+    }
+  }, [onBuyWithCredits, finalPrice, credits])
```

Comment:
Credits isn't being used here, should we keep it as a dependency?
```suggestion
  }, [onBuyWithCredits, finalPrice])
```

Reply:
good catch!